autogen/agentchat/conversable_agent.py

```diff
@@ -105,7 +101,9 @@ def __init__(
         self._oai_messages = defaultdict(list)
         self._oai_system_message = [{"content": system_message, "role": "system"}]
         self._is_termination_msg = (
-            is_termination_msg if is_termination_msg is not None else (lambda x: x.get("content") == "TERMINATE")
+            is_termination_msg
+            if is_termination_msg is not None
+            else (lambda x: content_str(x.get("content")) == "TERMINATE")
```

Comment:
That's right. 
For LLM, the content_str will do nothing and return the same results as the previous version.
For LMM, nothing is supported in the past. If the LMM follows the instruction correctly, it would return a single word "TERMINATE" and also be compatible with the code here. So, this new version should be fine too. 

We can defer the changes to the future.